### DISCUSSION TITLE
TODO

### DISCUSSION THREAD

**Discussion by cuisongliu** [labring-actions/cluster-image#106] [General]
- [x] 测试镜像
- [x] 整理命令（所有的判断看能不能统一一下）
- [x] 独立registry
- [x] 开发镜像和sealos的patch镜像合并
- [x] 判断最新的镜像是否是最新的release
- [x] 全局的环境变量看是否能统一配置，别挨个文件修改
- [x] k8s镜像开关（关闭就不能build镜像）
- [ ] 优化rootfs脚本
- [ ] 优化apps/configs

**Comment by cuisongliu** [comment]
```
labring/kubernetes:v1.24.4
labring/kubernetes:v1.24.4-4.1.1
labring/kubernetes-docker:v1.24.4
labring/kubernetes-docker:v1.24.4-4.1.1

labring/kubernetes-dev:v1.24
labring/kubernetes-docker:v1.24
```

**Comment by cuisongliu** [comment]
判断release的版本是否为最新版本，如果是才进行labring/kubernetes:v1.24.4 的tag否则 只出labring/kubernetes:v1.24.4-v4.1.1的版本

**Reply by muicoder** [reply]
https://github.com/labring/cluster-image/pull/123

**Comment by muicoder** [comment]
现阶段之后：
优化rootfs脚本
优化apps/configs
